freebsd/FreeBSDProcessList.c

```diff
@@ -7,6 +7,7 @@ in the source distribution for its full text.
 
 #include "config.h" // IWYU pragma: keep
 
+#include "freebsd/FreeBSDMachine.h"
```

Comment:
Header order, should go below `XUtils.h` as this is not paired with `FreeBSDMachine.c`.

On that note: `FreeBSDProcess.h` should be `freebsd/FreeBSDProcess.h` instead (sorted below `XUtils.h` too.